packages/scene-composer/src/augmentations/components/three-fiber/anchor/AnchorWidget.tsx

```diff
@@ -1,16 +1,19 @@
-import * as THREE from 'three';
+import { ThreeEvent, extend } from '@react-three/fiber';
 import React, { ReactElement, useCallback, useContext, useEffect, useMemo, useRef, useState } from 'react';
-import { extend, ThreeEvent } from '@react-three/fiber';
+import * as THREE from 'three';
 
 import {
+  CustomIconSvgString,
   ErrorIconSvgString,
   InfoIconSvgString,
   SelectedIconSvgString,
   VideoIconSvgString,
   WarningIconSvgString,
 } from '../../../../assets';
-import { WidgetSprite, WidgetVisual } from '../../../three/visuals';
-import { Anchor } from '../../../three';
+import { Layers } from '../../../../common/constants';
+import { sceneComposerIdContext } from '../../../../common/sceneComposerIdContext';
+import { traverseSvg } from '../../../../components/panels/scene-components/tag-style/ColorPicker/ColorPickerUtils/SvgParserHelper';
+import useTagSettings from '../../../../hooks/useTagSettings';
```

Comment:
(Not related to this PR): Have we considered utilizing module resolution in the `scene-composer` package to enable easy absolute imports? Some of the other packages, including the dashboard, have a setup to enable this:

```
import { Layers } from '~/common/constants';
import { sceneComposerIdContext } from '~/common/sceneComposerIdContext';
import { traverseSvg } from '~/components/panels/scene-components/tag-style/ColorPicker/ColorPickerUtils/SvgParserHelper';
import useTagSettings from '~/hooks/useTagSettings';
```

These relative imports are getting quite long! :p

Reply:
Yes. I will take a note of it to improve these imports.

Reply:
Can you send me a reference?

Reply:
https://github.com/awslabs/iot-app-kit/blob/main/packages/dashboard/tsconfig.json#L9